packages/tizen_bundle/lib/tizen_bundle.dart

```diff
@@ -0,0 +1,326 @@
+// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ffi';
+
+import 'package:ffi/ffi.dart';
+import 'package:tizen_interop/4.0/tizen.dart';
+import 'package:tizen_log/tizen_log.dart';
+
+/// The class for the property type of the bundle.
+class _BundleTypeProperty {
+  /// The array type.
+  static const int array = 0x0100;
+
+  /// The primitive type.
+  static const int primitive = 0x0200;
+
+  /// The measurable type.
+  static const int measurable = 0x0400;
+}
+
+/// The class for the type of the bundle.
+class BundleType {
```

Comment:
Please consider using `enum` for these cases. 
https://codewithandrea.com/tips/enums-with-members-dart-2.17

Reply:
Done. Thanks for sharing.